### DISCUSSION TITLE
9 Nov.2022 = A challenging day due to Sling service disruptions

### DISCUSSION THREAD

**Discussion by bradut** [GerryDazoo/Slinger#635] [General]
Just reflecting on the fact that exactly one year ago, on 9 Nov.2022 Sling as a service was retired. 
To be fair, they did give us a two-year heads up, but didn't mention that they would render **our** devices inactive. 

Fortunately, smart and skilled people (like Gerry and others) managed to find a solution to revive most boxes, and gave us Slinger, Slinger Player and Community Support .

**Comment by flipper2222222** [comment]
I second those sentiments, a marvellous and commendable effort by Gerry. 
much appreciated

**Reply by floormanager1** [reply]
A year later, still using my Slingboxes (two KIA though)
